mirage/fixtures/external-storage-services.ts

```diff
@@ -14,10 +23,38 @@ export default [
         maxUploadMb: 5,
         iconUrl: 'https://www.box.com/favicon.ico',
     },
+    {
+        id: 'bitbucket',
+        name: 'Bitbucket',
+        credentialsFormat: CredentialsFormat.OAUTH2,
+        termsOfService: [
+            TermsOfServiceCapabilities.FORKING,
+            TermsOfServiceCapabilities.PERMISSIONS,
+            TermsOfServiceCapabilities.REGISTERING,
+            TermsOfServiceCapabilities.FILE_VERSIONS,
+            TermsOfServiceCapabilities.LOGS,
+        ],
+        readOnly: false,
+        supportsCopy: true,
+        supportsUserSpecificRootFolder: true,
+        supportsFileVersioning: true,
+        supportsBulkDownload: true,
```

Comment:
Are these not instead strings on the ConfiguredStorateAddon's `connected_operation_names`?

Reply:
Ahh, yup. There's `readOnly` that should be covered by `ConfiguredStorageAddon.connectedCapabilities`, `supportsCopy` which should be covered by `ConfiguredStorateAddon.connectedOperationNames` as well. Will clean it up here and in the mirage fixture

Reply:
Are `maxConcurrentDownloads` and `maxUploadMb` real things that should stay?

Reply:
Yeah, those are still in the API Contract

Reply:
The model will need updating as well to remove those fields.